Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -506,6 +508,59 @@ public static Task TestRegistrationAsync(this UITestContext context, UserRegistr
                     .SearchForFeature(featureName).IsEnabled.Should.Equal(originalEnabledState));
             });
 
+    public static Task TestMediaOperationsAsync(this UITestContext context) =>
+        context.ExecuteTestAsync(
+            "Test media operations",
+            async () =>
+            {
+                await context.GoToRelativeUrlAsync("Admin/Media");
+
+                context.UploadSamplePngByIdOfAnyVisibility("fileupload");
+                context.UploadSamplePdfByIdOfAnyVisibility("fileupload");
+
+                // Workaround for pending uploads.
+                context.WaitForPageLoad();
+                await context.ClickReliablyOnAsync(By.CssSelector("body"));
+
+                context
+                    .Get(By.CssSelector("#mediaContainerMain tbody tr:nth-child(4) .break-word"))
+                    .Text
+                    .ShouldBeAsString("Image.png");
+                context
+                    .Get(By.CssSelector("#mediaContainerMain tbody tr:nth-child(2) .break-word"))
+                    .Text
+                    .ShouldBeAsString("Document.pdf");
```

Comment:
Use the values from `FileUploadHelper` instead of repeating the file names here.